prow/pjutil/abort_test.go

```diff
@@ -194,15 +194,15 @@ func TestTerminateOlderJobs(t *testing.T) {
 					},
 				},
 			},
-			terminateddPJs: sets.NewString("old", "older"),
+			terminateddPJs: sets.NewString(),
 		},
 		{
 			name: "terminate older jobs with different orders of refs",
 			pjs: []prowjobv1.ProwJob{
 				{
 					ObjectMeta: metav1.ObjectMeta{Name: "newest", Namespace: fakePJNS},
 					Spec: prowjobv1.ProwJobSpec{
-						Type: prowjobv1.BatchJob,
```

Comment:
Can we test that batch jobs do not get terminated?

Reply:
We do, the testcase above this that used to test batch job termination now verifys that they do not get terminated anymore.

Reply:
...But I just realized I didn't change the testname which was confusing, fixed that now